<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Towards Ancestry-Agnostic Genetics</title>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.css" integrity="sha384-n8MVd4RsNIU0tAv4ct0nTaAbDJwPJzDEaqSD1odI+WdtXRGWt2kTvGFasHpSy3SV" crossorigin="anonymous">
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.js" integrity="sha384-XjKyOOlGwcjNTAIQHIpgOno0Hl1YQqzUOEleOLALmuqehneUG+vnGctmUb0ZY0l8" crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/contrib/auto-render.min.js" integrity="sha384-+VBxd3r6XgURycqtZ117nYw44OOcIax56Z4dCRWbxyPt0Koah1uHoK0o4+/RRE05" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>

    <style>
        body {
            position: relative;
        }

        .container img {
            width: 100%;
            height: auto;
        }

        .container {
            margin: 0 auto;
            padding: 0 20px;
            max-width: 800px;
            margin-bottom: 50px; 
        }

        @media (min-width: 768px) {
            .container {
                max-width: 40%;
                margin-bottom: 300px;
            }
        }

        .top-link {
            display: block;
            text-align: center;
            font-size: 18px;
            text-decoration: none;
            color: blue;
            margin-top: 15px;
        }

        @media (min-width: 768px) {
            .top-link {
                font-size: 30px;
            }
        }

        .centered-italic {
            text-align: center;
            font-style: italic;
        }

        h1, h2 {
            text-align: center;
            margin: 20px 0;
        }

    </style>
        
</head>
<body>
    <a href="http://www.barneyhill.com" class="top-link">barneyhill.com</a>

    <div class="container">

        <h1>Towards Ancestry-Agnostic Genetics</h1>

        <h2>where we are</h2>
        <p>All summaries of statistical genetics begin the same: within the last 20 years we’ve seen enormous success in the aggregation of human genetic variation at the population scale with projects such as the Genome Aggregation Database (gnomAD) and the UK Biobank. Using this genetic variation we can then <strong>associate</strong> between disease and variation with Genome-Wide Association Studies (GWAS) and then <strong>predict</strong> disease-risk based on prior associations in the case of complex, non-mendelian diseases.</p>

        <p>What these summaries often neglect is that statistical genetics has largely failed to create methods that scalably associate and predict disease independent of an individual’s ancestry. In practice what this means is that disease-variant associations often fail to replicate across population cohorts, while polygenic risk score predictive accuracy drops when applied to ancestries not represented in the training data. Consequently, many current genetic studies tend to focus on participants from a specific ancestral background. My concern is that a failure to model the full diversity of human variation within disease association will further exacerbate existing healthcare disparities.</p>

        <p>In this essay I’ll introduce prior work in understanding the genetic architecture of complex disease alongside covering recent advances in modelling protein sequence likelihood using unsupervised machine-learning models. I will then introduce a methodology leveraging these advances that associates coding variation with disease at the protein-isoform-level across diverse ancestries.</p>

        <h2>what we’ve done</h2>
        <h3>GWAS</h3>
        <p>At the foundation of modern statistical genetics is the Genome-Wide Association Study (GWAS). This is a type of association test in which we gather a cohort of individuals and make associations between many variants within genetic variants and some measured trait. More specifically this is an (additive) linear model where \(X_i \in \{0, 1, 2\}\) represents a vector of individual’s allele counts and \(Y\) represents the individual’s recorded trait.</p>
        <p>\[
        Y=\sum_i \beta_i X_i +\epsilon
        \]</p>

        <p>Of course if we use this model we’ll quickly encounter a problem - we forgot our covariates! Across the vast majority of complex diseases, age and sex will account for by far the largest variance within disease prevalence so they will need to be accounted for.</p>
        <p>\[
        Y= \text{age} + \text{sex} + \sum_i \beta_i X_i + \epsilon 
        \]</p>

        <p>Another issue we will encounter is population stratification, manifesting as allele frequency disparities between cases and controls due to ancestry variations, can lead to false associations. <a href="https://www.nature.com/articles/ng1847">In 2006, the usage of genetic principal components were introduced in GWAS</a> to mitigate such inaccuracies, enhancing the power to discern authentic associations. Utilizing these components adjusts for the confounding effects of population stratification, bolstering the validity of observed associations.</p>
        <p>\[
        Y=\text{age} + \text{sex} + \sum_j \text{PC}_j + \sum_i \beta_i X_i + \epsilon 
        \]</p>

        <p>Over the last 20 years the GWAS model in the above form has been <a href="https://www.ebi.ac.uk/gwas/home">successfully used to explore phenotype-genome associations across thousands of traits</a>. Despite this much of the genetic contribution to complex diseases remains unexplained. One common explanation for this phenomenon is that the standard GWAS only has sufficient power to detect the most common variants. As potentially damaging variants are rare in populations due to natural selection, the identification of rare variation becomes particularly important.</p>

        <h3>Gene-burden association testing</h3>
        <p>A simple method to integrate rare variant information into association testing is the gene burden test. Instead of associating at the variant-level, the gene burden test seeks to accumulate all variants \(v_j\) within a gene for which we have some prior of pathogenicity \(w_j\). The motivation for testing at the gene-level is that exome-sequencing data captures protein-coding regions where damaging variation often has the same effect of disrupting the protein corresponds to the gene’s function.</p>
        <p>\[
        Q_\text{burden} = (\sum_{j=1}^q w_j v_j)^2
        \]</p>

        <p>While this test is successful in integrating variants across the allele frequency spectrum, we still encounter the same fundamental issue of not considering epistasis. In this context epistasis refers to a variant having different effects depending on the other variants present within the gene. An example of this is intragenic suppression wherein a deleterious variant is suppressed by the presence of a second nearby variant. Within coding regions this is often due to variants corresponding to amino acids interacting within a protein.</p>

        <p>Not considering epistasis becomes problematic when we begin to associate across genetically diverse populations. In the case of a genetically homogenous population we can expect the local context of a variant to be relatively similar - variant effect will be consistent across individuals, but as we expand our cohort we will see a variant have varied effects based on different local contexts. This causes loss in the statistical power to detect associations at the variant level without restriction to a cohort with a related ancestral background.</p>

        <img src="https://github.com/barneyhill/www/raw/main/assets/snps.png" />

        <p class="centered-italic">Genotype Variation at IGF1 (<a href="https://pga.gs.washington.edu/finished_genes.html">https://pga.gs.washington.edu/finished_genes.html</a>)</p>

        <h3>Unsupervised protein sequence likelihood modeling</h3>

        <p>Within the last few years, methods such as <a href="https://www.nature.com/articles/s41586-021-04043-8">EVE</a>, <a href="https://proceedings.mlr.press/v162/notin22a/notin22a.pdf">Transception</a>, and <a href="https://www.nature.com/articles/s41588-023-01465-0">ESM1b</a> have been released leveraging evolutionary information from multiple sequence alignments leading to significant advances in predicting the fitness impact of protein variants. These models rely on the insight that patterns of variation conserved across diverse species provide the information required for expressive models of the probability distribution \(p(x)\) of a given protein sequence \(x\). Given the ability to model \(p(x)\), we can then estimate the effect \(\sigma_v\) of a variant sequence \(x_v\) where \(\sigma_v = \log\left(\frac{p(x_v)}{p(x)}\right)\). A less probable variant sequence corresponds to a less fit and likely more pathogenic sequence.</p>

        <p>These methods have been used to estimate all ~450 million possible missense variant effects within the human genome. For example, in a recent paper, ESM1b was shown to outperform existing pathogenicity prediction methods and then showed that ~2 million variants are classified as damaging only in specific protein isoforms. Current convention within variant annotation for association testing is to ignore any specific protein isoforms and only consider the ‘canonical’ sequence.</p>

        <h2>what happens next</h2>

        <p>Building upon the foundations discussed, I believe there is potential in associating an individual’s specific protein-isoform fitness with disease. This approach has the potential advantages:</p>

        <ul>
            <li>Protein sequence likelihood models account for epistatic interactions between variants on a individual's protein sequence providing portability across diverse ancestries by not relying on a reference sequence.</li>
            <li>By testing at the protein-isoform level, we can detect pathogenic variation that would be benign on the canonical protein and vice-versa.</li>
        </ul>

        <p>To formalize this, consider an individual \(i\) and a specific protein isoform \(j\). We have the isoform sequence \(x_{ij}\) and a continuous phenotype value \(y_i\). Taking the previous equation for a variant effect as \(\sigma_v = \log\left(\frac{p(x_v)}{p(x)}\right)\), we have \(\sigma_{ij} = \log \left( \frac{p(x_{ij})}{ p(x_j) }\right)\) where \(x_j\) is a protein isoform’s reference sequence. Using linear regression, we model the continuous phenotype value \(y_i\) as a function of the predicted protein-isoform effect \(\sigma_{ij}\) generated from a model such as ESM1b or Transception:</p>

        <p>\[
        \begin{align*}
        y_i &= \beta_0 + \beta_1\sigma_{ij}+\epsilon_i \\
        y_i &= \beta_0 + \beta_1 \log \frac{p(x_{ij})}{p(x_j)}+\epsilon_i \\
        y_i &= \beta_0 + \beta_1(\log(p(x_{ij})) - \log(p(x_j)))+\epsilon_i \\
        y_i &= \beta_0 + \beta_1\log(p(x_{ij})) - \beta_1\log(p(x_j)) +\epsilon_i \\
        y_i &= (\beta_0 - \beta_1\log(p(x_j))) + \beta_1\log(p(x_{ij})) +\epsilon_i \\
        \end{align*}
        \]</p>

        <p>We can treat the term \((\beta_0 - \beta_1\log(p(x_j)))\) as an intercept as it is a constant across all individuals, hence controlling for common factors we have:</p>

        <p>\[
        y_i = \beta_0' + \beta_1 \log(p(x_{ij})) + \beta_2  \text{age}_i + \beta_3  \text{sex}_i + \beta_4 \sum_k \text{PC}_k
        \]</p>

        <p>We can then test the null hypothesis \(H_0: \beta_1 = 0\) against the alternative hypothesis \(H_a: \beta_1 \neq 0\) accounting for multiple test correction. This formula relies only on computing \(p(x_{ij})\) and so is conveniently reference-free.</p>
    </container>
</body>
</html>
